### PR TITLE
Fix issues on code review

### PR DESCRIPTION
**Pull request by Sathiyakugan**

### CODE REVIEW

File: modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/InterceptorConfigReader.java [hunk 1]
```diff
@@ -29,16 +29,20 @@
...
 
     /**
-     * Reads the configs in the class resource
-     * "instrumentation-config.json".
+     * Reads the configs in the class resource.
+     * As per the documentation have to pass the Method signature in binary format.
+     * use this link https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html to add the binary format.
      *
-     * @return
+     * @return result
```
Comment (darshanasbg) [inline modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/InterceptorConfigReader.java:36]: Be descriptive

Comment (Sathiyakugan) [reply modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/InterceptorConfigReader.java]: Fixed

File: modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/MethodInfoConfig.java [hunk 2]
```diff
@@ -22,40 +22,67 @@
...
     public String getSignature() {
 
         return signature;
     }
 
+    /**
+     *This method is to Verify the Method with the Class loaded and the Method in the config.
+     *
+     * @param methodName
+     * @param signature
```
Comment (darshanasbg) [inline modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/MethodInfoConfig.java:83]: Be descriptive

File: modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/MethodInfoConfig.java [hunk 3]
```diff
@@ -22,40 +22,67 @@
...
+     *
+     * @return
+     */
     public String getSignature() {
 
         return signature;
     }
 
+    /**
+     *This method is to Verify the Method with the Class loaded and the Method in the config.
```
Comment (darshanasbg) [inline modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/MethodInfoConfig.java:80]: Have a space after the *

**Pull request merged**